pkg/parser/model/ddl.go

```diff
@@ -1138,6 +1138,12 @@ type SchemaDiff struct {
 	SchemaID int64      `json:"schema_id"`
 	TableID  int64      `json:"table_id"`
 
+	// ActionTypes is the list of action types done together within a multiple schema
+	// change job. As the job might contain multiple steps that changes schema version,
+	// if some step only contains one action, Type will be that action, and ActionTypes
+	// will be empty.
+	// for other types of job, it will always be empty.
+	ActionTypes []ActionType `json:"action_types"`
```

Comment:
Using `SubTypes` instead of `ActionTypes`? Thus unified with the `Type` field and `MultiSchemaInfo.SubJobs`. Just a suggestion